src/test_workflow/integ_test/service_opensearch.py

```diff
@@ -47,6 +48,16 @@ def start(self) -> None:
         self.opensearch_yml_path = self.dist.config_path
         self.security_plugin_dir = os.path.join(self.install_dir, "plugins", "opensearch-security")
 
+        # Additional for sql
+        print("start catalog")
+        self.catalog_json_path = os.path.join(self.install_dir, "catalog.json")
+        os.system(f"curl -SL https://raw.githubusercontent.com/opensearch-project/sql/2.4/integ-test/src/test/resources/catalog/catalog.json -o {self.catalog_json_path}")
+        subprocess.check_call(
+            f"echo y | {self.dist.keystore_cmd} add-file plugins.query.federation.datasources.config catalog.json",
+            shell=True,
+            cwd=self.install_dir,
+        )
```

Comment:
Can we do this is generic way? More plugins would need some or other things for their integration test. I believe SQL should add this via their gradle tasks (if possible).
Also what about previous versions (1.x series) that do not need this?
Something like this https://github.com/opensearch-project/opensearch-build/blob/main/manifests/2.5.0/opensearch-2.5.0-test.yml#L16-L17 I believe is the right way to go forward